### ISSUE TITLE
Error when reacting with an emoji

### ISSUE THREAD

**Issue by MichaelPriebe** [hydrabolt/discord.js#1152]
I get TypeError: Cannot read property 'name' of undefined when reacting like so:

message.react('💩');

```
TypeError: Cannot read property 'name' of undefined
    at new MessageReaction ([snip]\node_modules\discord.js\src\structures\MessageReaction.js:34:48)
    at Message.patch ([snip]\node_modules\discord.js\src\structures\Message.js:231:34)
    at MessageUpdateAction.handle ([snip]\node_modules\discord.js\src\client\actions\MessageUpdate.js:14:17)
    at rest.makeRequest.then.data ([snip]\node_modules\discord.js\src\client\rest\RESTMethods.js:96:55)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```
space (windows 7, #master, github name: spaceeec, node 7.4) also gets this error.

It looks 100% fine, the message gets the reaction, but it still shows that error.

**Comment by Programmix** [comment]
This is a duplicate of #1071.
It should have been fixed by PR #1137.
Please update your discord.js version and this issue should be fixed; if it is not, let us know.

**Comment by Gawdl3y** [comment]
It's fixed in master, but hasn't been released yet.
